src/code-templates/services/order-service/jest.config.js

```diff
@@ -99,7 +99,20 @@ module.exports = {
   // projects: undefined,
 
   // Use this configuration option to add custom reporters to Jest
-  // reporters: undefined,
+  reporters: [
+    "default",
+    [
+      "@jest-performance-reporter/core",
+      {
+        "errorAfterMs": 1000,
```

Comment:
I would suggest that in typical backend testing (not E2E), if a test lasts more than 70ms - It should be at least a warning

Reply:
We can do a warning for 70ms+.
An error in 500ms+?

Do you have data to support the decision of the outliers or is it from your experience?